dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/services/DirectoryLoadService.java

```diff
@@ -0,0 +1,88 @@
+package nl.tudelft.dnainator.javafx.services;
+
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.StreamSupport;
+
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.concurrent.Service;
+import javafx.concurrent.Task;
+
+/**
+ * A JavaFX service that loads all the directories from a specific path.
+ */
+public class DirectoryLoadService extends Service<ObservableList<String>> {
+	private static final String DEFAULT_DB_PATH = "target" + File.separator + "db";
```

Comment:
For example, if you run this on Linux, you get `~/target/`. Not really meaningful.